config.json

```diff
@@ -35,6 +35,10 @@
                     "ovmcontrol": "/path/to/ovmcontrol"
                 }
             ]
+        },
+        "exadata": {
+            "enabled": false,
+            "notrootuser": "ercole"
```

Comment:
exadata require root permissio

Reply:
Yes, it require root permission, so..?

Reply:
The default value should be root or "", not ercole

Reply:
Done :+1: